MonoGame.Framework/Platform/Windows/WinFormsGameWindow.cs

```diff
@@ -240,7 +240,7 @@ private void UnregisterFromAllWindows()
 
         private void OnFormClosing(object sender, FormClosingEventArgs e)
         {
-            e.Cancel = true;
+            // TODO WHY? e.Cancel = true;
```

Comment:
@ThomasFOG, @SimonDarksideJ, @Jjagg Took a bit longer to track down, but this is what was stopping the Form from closing and `Dispose()` from being called. I can't see a reason why we'd want to cancel the form closing. Does anyone else know?